scripts/create-clusters.sh

```diff
@@ -75,16 +75,17 @@ function configure-insecure-registry-and-reload() {
 }
 
 function insecure-registry-config-cmd() {
-  if test -f "${docker_daemon_config}"; then
-    sed -i "1a \    \"insecure-registries\": [\"${CONTAINER_REGISTRY_HOST}\"]" ${docker_daemon_config}
-  else
-    echo "cat <<EOF > ${docker_daemon_config}
+  echo "
+    if test -f ${docker_daemon_config}; then
```

Comment:
It does not suffice to just check it on the host because it will assume the host's existence of `/etc/docker/daemon.json` when inside the `clusterX-control-plane` docker containers it does not exist. So this check needs to be done on the host AND inside the cluster because it's called from 2 different contexts.